slides/slides/1.1-getting-started/assets/python-interpreter.drawio

```diff
@@ -0,0 +1,82 @@
+<mxfile host="Electron" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/26.0.16 Chrome/132.0.6834.196 Electron/34.2.0 Safari/537.36" version="26.0.16">
```

Comment:
curiosity, are you rendering .drawio directly or is this the same `svg` below?
I was thinking of committing the `.drawio.png` (editable bitmap image) for one drawio image of mine but still unsure.

Reply:
I'm rendering the `.svg` file but also committed the `.drawio` version because I thought that it was needed for simple editing, but it seems that the `.svg` file exported by Draw.io already contains all the information so the `.drawio` is redundant. I'll delete it.